src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Color.values().length);
```

Comment:
```suggestion
    private final Random random = new Random();
```
int index = random.nextInt(Color.values().length) - this line should be moved inside of getRandomColor